css/layout.css

```diff
@@ -32,6 +33,26 @@
   color: white;
 }
 
+ul {
+  padding: 0;
+}
+
+li {
+  list-style-type: none;
+}
```

Comment:
These two should go in your `styles.css`. I would just put utility classes in the `layout.css` file.